### PR TITLE
[docs][cloud] nodejs application for cloud

### PR DESCRIPTION
**Pull request by ddhodge**
node.js quickstart application for Coud

@netlify /latest/yugabyte-cloud/cloud-quickstart/cloud-build-apps/

### CODE REVIEW

File: docs/content/latest/yugabyte-cloud/cloud-quickstart/cloud-build-apps/cloud-ysql-node.md [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+    parent: cloud-build-apps
+    name: Node.js
+    identifier: cloud-node
+    weight: 400
+type: page
+isTocNested: true
+showAsideToc: true
+---
+
+The following tutorial shows a small [Node.js application](https://github.com/yugabyte/yugabyte-simple-node-app) that connects to a YugabyteDB cluster using the [node-postgres module](../../../../reference/drivers/ysql-client-drivers/#node-postgres) and performs basic SQL operations. Use the application as a template to get started with Yugabyte Cloud in Node.js.
```
Comment (aishwarya24) [inline docs/content/latest/yugabyte-cloud/cloud-quickstart/cloud-build-apps/cloud-ysql-node.md:16]: You can also link this Build a Node.js page in this reference section : https://docs.yugabyte.com/latest/reference/drivers/ysql-client-drivers/#node-postgres

File: docs/content/latest/yugabyte-cloud/cloud-quickstart/cloud-build-apps/cloud-ysql-node.md [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+
+## Prerequisites
+
+In addition to the latest version of [Node.js](https://nodejs.org/en/download/), this tutorial requires the following.
+
+### Yugabyte Cloud
+
+- You have a cluster deployed in Yugabyte Cloud. To get started, use the [Quick start](../../).
+- You downloaded the cluster CA certificate. Refer to [Download your cluster certificate](../../../cloud-secure-clusters/cloud-authentication/#download-your-cluster-certificate).
```
Comment (aishwarya24) [inline docs/content/latest/yugabyte-cloud/cloud-quickstart/cloud-build-apps/cloud-ysql-node.md:25]: You could probably mention that the CA certificate must be downloaded for YSQL.

**Review by aishwarya24 — APPROVED** [review]

File: docs/content/latest/yugabyte-cloud/cloud-quickstart/cloud-build-apps/cloud-ysql-node.md (hunk 2 above)
Comment (ddhodge) [reply docs/content/latest/yugabyte-cloud/cloud-quickstart/cloud-build-apps/cloud-ysql-node.md:25]: YSQL or YCQL, same certificate

File: docs/content/latest/yugabyte-cloud/cloud-quickstart/cloud-build-apps/cloud-ysql-node.md (hunk 1 above)
Comment (ddhodge) [reply docs/content/latest/yugabyte-cloud/cloud-quickstart/cloud-build-apps/cloud-ysql-node.md:16]: Haven't yet decided what will happen with the older examples, they may be replaced by this one - which works equally well for local install

**Pull request merged**